demo/site/src/documents/pages/blocks/ColumnsBlock.tsx

```diff
@@ -33,11 +33,19 @@ const ColumnsContentBlock = withPreview(
     { label: "Columns" },
 );
 
+const layoutToStyleMap: { [key: string]: string } = {
+    "6-12-6": styles["layout-6-12-6"],
+    "4-16-4": styles["layout-4-16-4"],
+    "9-9": styles["layout-9-9"],
+    "12-6": styles["layout-12-6"],
+    "6-12": styles["layout-6-12"],
+};
```

Comment:
This is only necessary because the columns block doesn't use an enum for layout. We could change this in a future PR.